vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -66,8 +66,11 @@
         AVAILABLE("available"),
         @SerializedName("uploading")
         UPLOADING("uploading"),
+        @SerializedName("transcode_starting")
+        TRANSCODE_STARTING("transcode_starting"),
         @SerializedName("transcoding")
```

Comment:
Kind of can go either way since these strings shouldn't be used anywhere else and also don't necessarily need to be tied to one another.
But it looks prettier: b26ec0e